deeplearning4j-nlp-parent/deeplearning4j-nlp-japanese/src/main/java/org/deeplearning4j/text/tokenization/tokenizerfactory/JapaneseTokenizerFactory.java

```diff
@@ -1,23 +1,45 @@
 package org.deeplearning4j.text.tokenization.tokenizerfactory;
 
+import org.deeplearning4j.text.tokenization.tokenizer.JapaneseTokenizer;
 import org.deeplearning4j.text.tokenization.tokenizer.TokenPreProcess;
 import org.deeplearning4j.text.tokenization.tokenizer.Tokenizer;
-import org.deeplearning4j.text.tokenization.tokenizer.JapaneseTokenizer;
 
 import java.io.InputStream;
 
```

Comment:
Javadoc mentioning the thread safeness and how the kuromoji instance works would make sense here.